### ISSUE TITLE
done() invoked with non-Error

### ISSUE THREAD

**Issue by rrodriguesdashlane** [Codeception/CodeceptJS#66]
Hi guys,

I got an issue when using grabTextFrom followed by a click which always return the following error `done() invoked with non-Error: {"status":0,"sessionId":"9458f069-1907-4362-b66f-012a6259a75d","state":"success","value":null,"class":"org.openqa.selenium.remote.Response","hCode":385625253}`

**Example:**
```
Scenario.only ('scenario test', function*(I) {
	I.amOnPage('my url');
  	var checked = yield I.grabTextFrom('input[type="password"]');
  	I.click('Create my account');
});
```

Is there anything I'm doing wrong in my example? Did anyone already face the same issue?

Thanks,

**Comment by DavertMik** [comment]
This can be a bug. Thanks for reporting, I will try to reproduce it

**Comment by DavertMik** [comment]
I hope this should be fixed in 0.3.0 :)
